app/inputs/datetime_picker_input.rb

```diff
@@ -0,0 +1,41 @@
+class DatetimePickerInput < SimpleForm::Inputs::DateTimeInput
+  include ActionView::Helpers::TagHelper
+  include ActionView::Context
+
+  def initialize(*args)
+    super(*args)
+
+    # Format value as iso8601 for javascript new Date() compatibility
+    if object && (value = object.send(attribute_name))
+      input_html_options[:value] = value.iso8601
+    end
+  end
+
+  def input(wrapper_options)
```

Comment:
Unused method argument - `wrapper_options`. If it's necessary, use `_` or `_wrapper_options` as an argument name to indicate that it won't be used. You can also write as `input(*)` if you want the method to accept any arguments but don't care about them.